src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.ts

```diff
@@ -86,8 +91,8 @@ export const getIndicesViaSearch = async ({
       },
     },
   })
+    .pipe(filter((resp) => isCompleteResponse(resp) || !!isErrorResponse(resp)))
```

Comment:
I don't think this logic is correct...
A partial response is not an error response, so it will pass through the filter.

Also, no reason to use a separate `pipe` for each operator.
They are chainable. ;-)